systemtest/src/test/java/io/strimzi/systemtest/watcher/AbstractNamespaceST.java

```diff
@@ -4,67 +4,326 @@
+    final void testUOWatchingOtherNamespace(ExtensionContext extensionContext) {
+        final TestStorage testStorage = new TestStorage(extensionContext);
+
+        LOGGER.info("Creating KafkaUser: {}/{} in namespace different from on Kafka Cluster or Cluster Operator are located in", PRIMARY_KAFKA_WATCHED_NAMESPACE, testStorage.getUsername());
+        resourceManager.createResource(extensionContext, KafkaUserTemplates.tlsUser(PRIMARY_KAFKA_WATCHED_NAMESPACE, PRIMARY_KAFKA_NAME, testStorage.getUsername()).build());
+        LOGGER.info("KafkaUser: {}/{} was successfully created", PRIMARY_KAFKA_WATCHED_NAMESPACE, testStorage.getUsername());
+    }
+
+    /**
+     * @description This test case verifies that KafkaUser custom resource managed by is act upon by User Operator from correctly, despite being watched
+     * from different namespace.
+     *
+     * @steps
+     *  1. - As part of setup Kafka Cluster is deployed in main namespace, with the User Operator configured to watch other namespace.
+     *     - Kafka and its components are deployed and ready.
+     *  2. - KafkaUser custom resource is created in namespace watched by Topic Operator.
+     *     - Topic Operator acts upon KafkaUser custom resource which is transitioned into ready state while also creating all other resources (e.g., Secret).
+     *  3. - Credentials generated due to this KafkaUser custom resources are used in order to allow clients to communicate with Kafka Cluster.
+     *     - Clients are able to successfully communicate with the Kafka Cluster.
+     *
+     * @usecase
+     *  - namespaces
+     *  - user-operator-watcher
+     */
+    @IsolatedTest
+    final void testUserInNamespaceDifferentFromUserOperator(ExtensionContext extensionContext) {
+        final TestStorage testStorage = new TestStorage(extensionContext, MAIN_TEST_NAMESPACE);
+
+        LOGGER.info("Creating KafkaUser: {}/{} residing in separated namespace, which is watched by Kafka located in Namespace: {}", PRIMARY_KAFKA_WATCHED_NAMESPACE, testStorage.getUsername(), MAIN_TEST_NAMESPACE);
+        KafkaUser user = KafkaUserTemplates.tlsUser(PRIMARY_KAFKA_WATCHED_NAMESPACE, PRIMARY_KAFKA_NAME, testStorage.getUsername()).build();
+        resourceManager.createResource(extensionContext, user);
+
+        Condition kafkaCondition = KafkaUserResource.kafkaUserClient().inNamespace(PRIMARY_KAFKA_WATCHED_NAMESPACE).withName(testStorage.getUsername())
+            .get().getStatus().getConditions().get(0);
+        LOGGER.info("KafkaUser condition status: {}", kafkaCondition.getStatus());
+        LOGGER.info("KafkaUser condition type: {}", kafkaCondition.getType());
+
+        assertThat(kafkaCondition.getType(), is(Ready.toString()));
+
+        LOGGER.info("Finding and Copying Secrets related to created KafkaUser into Namespace: {} which holds Kafka", testStorage.getNamespaceName());
+        List<Secret> secretsOfSecondNamespace = kubeClient(PRIMARY_KAFKA_WATCHED_NAMESPACE).listSecrets();
```

Comment:
```suggestion
        List<Secret> secretsOfSecondNamespace = kubeClient().listSecrets(PRIMARY_KAFKA_WATCHED_NAMESPACE);
```
By this we will be sure that the secrets are really listed from the provided namespace and not the context.

Reply:
I am not sure if I am following here, I need to somehow list those secrets, did you mean ? 
```java
kubeClient(PRIMARY_KAFKA_WATCHED_NAMESPACE).listSecrets(PRIMARY_KAFKA_WATCHED_NAMESPACE)
``` 
?

Reply:
```
public List<Secret> listSecrets(String namespaceName) {
        return client.secrets().inNamespace(namespaceName).list().getItems();
    }
```
you don't need to specify the namespace inside the kubeclient directly, as it always checks the namespace if specified